### DISCUSSION TITLE
Upgrading from 4.6 to 4.8

### DISCUSSION THREAD

**Discussion by ghubz** [WeblateOrg/weblate#6465] [Q&A]
Hi,

I am currently running weblate 4.6.2 and want to upgrade to 4.8

On the website it only mentions changes from 4.6 to 4.7:

> Notable configuration or dependencies changes:
> - There are several changes in settings_example.py, most notable middleware changes (MIDDLEWARE), please adjust your settings accordingly.
> - The DeepL machine translation now has a generic MT_DEEPL_API_URL setting to adapt to different subscription models more flexibly. The MT_DEEPL_API_VERSION setting is no longer used.
> - Django 3.2 is now required.

Is there anything else I need to know?

Thanks

**Comment by nijel** [comment]
No, the update inside major version work just fine. There are no special instructions for 4.8, that's why it's not in the docs.
